bin/gather-files

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env ruby
+
+##
+#
+# This script is used to clone and init all the git repositories atlantis-aqurium
+# needs in order to run "atlantis-aquarium build"
+# 
+# All repos will be placed under $HOME/repos folder 
+#
+##
+
+$repos_home = File.expand_path('~') + "/repos"
```

Comment:
Can this use $ENV["ATLANTIS_REPO_ROOT"] if it's defined?  It's nice not to force a particular organization on anybody using this.